dockers/base-xla/Dockerfile

```
@@ -115,4 +115,4 @@ RUN \
     conda info && \
     pip list && \
     python -c "import sys; assert sys.version[:3] == '$PYTHON_VERSION', sys.version" && \
-    python -c "import torch; ver = '$XLA_VERSION' ; ver = dict(nightly='1.8').get(ver, ver) ; assert torch.__version__[:3] == ver, torch.__version__"
+    python -c "import torch; ver = '$XLA_VERSION' ; ver = dict(nightly='1.9').get(ver, ver) ; assert torch.__version__[:3] == ver, torch.__version__"
```

I don't understand what this does...

`dict(nightly='1.9').get(ver, ver)`

How does this ever return anything different than `"$XLA_VERSION"`?

---

this is our consistency check that you have really installed what you requested
the mapping trick is because you download it nightly without an explicit version number but after installation, it has a specific number, in this case when you download nightly you get 1.9

---

got it.

maybe this would be simpler

```suggestion
    python -c "import torch; ver = '$XLA_VERSION' ; ver = '1.9' if ver == 'nightly' else ver ; assert torch.__version__[:3] == ver, torch.__version__"
```